source/ui/REIXS/REIXSSampleChamberButtonPanel.cpp

```diff
@@ -41,6 +42,7 @@ REIXSSampleChamberButtonPanel::REIXSSampleChamberButtonPanel(QWidget *parent) :
 	initializeUiComponents();
 
 	connect(stopAll_, SIGNAL(clicked()), this, SLOT(onStopButtonClicked()));
+	connect(stopAll2_, SIGNAL(clicked()), this, SLOT(onStopButtonClicked()));
```

Comment:
What is a stopAll2 button and how is it different from stopAll_?  Can it be more specific?  Also, why does it go to the same slot?